dehydrated

```diff
@@ -165,6 +167,16 @@ load_config() {
    done
   fi
 
+  # Check if we are running & are allowed to run as root
+  if [[ ! -z "$DEHYDRATED_USER" && $EUID == 0 ]]; then
+    if [ ! -z "$DEHYDRATED_GROUP" ]; then
+        group="-g $DEHYDRATED_GROUP"
+    fi
+    echo "# INFO: Running $0 as $DEHYDRATED_USER"
+    su -c $0 $group "$DEHYDRATED_USER"
```

Comment:
`$0` and `$group` aren't escaped. I assume that `$group` wasn't escaped to provide the `-g` flag and the (safe) group name as individual arguments.

Reply:
$group is on purpose, and I saw little point in escaping $0, but I probably should, for good measure.